project/Build.scala

```diff
@@ -226,19 +226,29 @@ object StorehausBuild extends Build {
     parallelExecution in Test := false
   ).dependsOn(storehausAlgebra % "test->test;compile->compile")
 
+  lazy val storehausLevelDB = module("leveldb").settings(
+    libraryDependencies +=
+      "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8",
+    parallelExecution in Test := false,
+    // workaround because of how sbt handles native libraries
+    // http://stackoverflow.com/questions/19425613/unsatisfiedlinkerror-with-native-library-under-sbt
+    testOptions in Test := Seq(),
+    fork in Test := true
+  ).dependsOn(storehausAlgebra % "test->test;compile->compile")
```

Comment:
Yup, I think that's what I meant to do when I did [this](https://github.com/twitter/storehaus/pull/258/files#diff-215113124f1de02f228327ba7abb45f7R251).